src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -125,6 +127,7 @@ static string LocalAddress(IBuilder builder)
             return builder.Build<ReadOnlySettings>().LocalAddress();
```

Comment:
not really related to your changes, but could we use context.Settings here instead?